docs/user-guide/add-ai-assistant.md

```diff
@@ -0,0 +1,50 @@
+# Add Gen AI Assistant
+
+KubeRocketCI portal allows you to integrate Large Language Models (LLMs for short) to help you observe the overall platform status, as well as troubleshoot some issues related to the platform. Assistant's configuration in performed via the Configuration tab, in the Gen AI section.
```

Comment:
We have two options: First, add a chat assistant to provide platform-related information and assist with any questions. Second, enable integration with CodeMie, which will onboard the application into both the platform and CodeMie if the checkbox is selected.

Reply:
Rephrased